lesson_11/homework10.py

```diff
@@ -0,0 +1,41 @@
+"""

```

Comment:
Ok main idea for these tests:
- read last file line after certain operation and parse it for correct statuses , username , log_level etc
- you could skip log exist check logic